cost-analyzer/values.yaml

```diff
@@ -1078,6 +1078,32 @@ kubecostAggregator:
     #   containerPort: 40000
   securityContext: {}  # Define a securityContext for the aggregator pod. This will take highest precedence.
 
+## Kubecost Diagnostic Pod (beta, disabled by default until GA) Enables the
+## Kubecost primary to report health/diagnostics of all agent clusters. Each
+## agent cluster sends its health/diagnostic data to a storage bucket. Future
+## versions may include repairing & alerting from the primary.
+## Ref: https://docs.kubecost.com/install-and-configure/install/diagnostics
+##
+diagnostics:
+  enabled: true
```

Comment:
This doesn't look disabled by default.

Reply:
We had some conversations with product and engineering and the thought was launch and learn.

I do feel that the risk is low and the potential benefit is high.

Reply:
Can I suggest updating the values comment then so it's not confusing?